DNN Platform/Library/Entities/Host/Host.cs

```diff
@@ -1223,7 +1223,7 @@ public static bool ShowCriticalErrors
         ///   Defaults to 1
         /// </remarks>
         /// -----------------------------------------------------------------------------
-        [Obsolete("Deprecated in 8.0.0")]
+        [Obsolete("Deprecated in 8.0.0. Scheduled removal in v11.0.0.")]

```

Comment:
Not sure if the site log is removed or just methods replace, but some basic explanation should be given to developers on what to do. Same for the others in this namespace